tests/aws/services/stepfunctions/templates/base/statemachines/wait_10_sec_and_pass.json5

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
This is probably not needed as it is a duplicate of `WAIT_1_MIN`, which can easily be reconfigured as it's done in tests such as `aws.services.stepfunctions.v2.base.test_base.TestSnfBase.test_event_bridge_events_base`. Should save us from having too many templates.